test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -252,11 +252,12 @@ def _run_dependency_check_test(image, ec2_connection, processor):
                         .get("severity", "UNKNOWN"))
```

Comment:
I looked at the logs, and it looks like `"Failed to load NIST data"` is never printed, even though `severity` has not been assigned any value.

This means that there was no `Exception`, but the `response.status_code` was not `200`.

Could you add an `else` condition to catch this and throw an Exception?

Reply:
done